### PR TITLE
Changelog for 1.3.0-beta2

### PR DESCRIPTION
**Pull request by sergeybykov**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -4,20 +4,116 @@ All notable end-user facing changes are documented in this file.
...
+  - Relational Storage: Added SiloName to Membership table #1787
+  - Added option to perform provider commands quietly #1762
+  - Fix for strange bug in BondSerializer #1790
+  - CreateOrleansTables_SqlServer.sql: Removed support for SQL Server 2000 and 2005 #1779
+  - Added support for using Consul 0.6.0 as a Membership Provider #1267
+  - Allow for extraneous keys in OrleanQuery table #1752
+- Streaming
+  - EventHub receiver metrics reporting abstraction #2146
+  - EventHub stream provider time based message purge #2093
+  - Upgrade to ServiceBus 3.3.2 and related changes. #2087
```
Comment (jdom) [inline changelog.md]: "Upgrade to ServiceBus 3.3.2 and related changes" is probably not valuable

File: changelog.md [hunk 2]
```diff
@@ -4,20 +4,116 @@ All notable end-user facing changes are documented in this file.
...
+  - Add Memory Stream Provider #2063
+  - Persistent stream pulling agent now uses exponential backoff #2078
+  - Add dynamic adding / removing stream providers functionality. #1966
+  - EventHub stream provider logging/monitoring #1857
```
Comment (jdom) [inline changelog.md]: I'd combine this with EventHub receiver metrics reporting abstraction #2146

File: changelog.md [hunk 3]
```diff
@@ -4,20 +4,116 @@ All notable end-user facing changes are documented in this file.
...
+  - Updating EventHubAdapterFactory for extensibility. #1861
+  - Consistent implicit subscription Id generation. #1828
+  - ServiceBus upgrade to 3.2.2 #1813
```
Comment (jdom) [inline changelog.md]: dupe (and not that useful)

File: changelog.md [hunk 4]
```diff
@@ -4,20 +4,116 @@ All notable end-user facing changes are documented in this file.
...
+  - Event hub stream provider EventData to cached data mapping #1727
+  - Refactor of EventHubStreamProvider to better support plugable CacheDataAdapter #1714
```
Comment (jdom) [inline changelog.md]: Several EventHub provider refactoring and making it extensible that should probably be summarized into 1 item only

File: changelog.md [hunk 5]
```diff
@@ -4,20 +4,116 @@ All notable end-user facing changes are documented in this file.
...
+- Bug fixes
+  - Fix SerializationManager.GetFallbackSerializer() in .NET Standard #2211
```
Comment (jdom) [inline changelog.md]: not relevant for this release

File: changelog.md [hunk 6]
```diff
@@ -4,20 +4,116 @@ All notable end-user facing changes are documented in this file.
...
+  - Fix bug in GrainFactoryServiceProvider #2201
```
Comment (jdom) [inline changelog.md]: not relevant, since this was not included in previous releases

File: changelog.md [hunk 7]
```diff
@@ -4,20 +4,116 @@ All notable end-user facing changes are documented in this file.
...
+  - CodeGen: fix generated DeepCopy method to call RecordObject earlier #2135
+  - Fix support for serializing SByte[] #2140
+  - Fix synchronization bug in Orleans/Async/BatchWorker #2133
+  - Fix #2119 by allowing full uninitialization in SiloHost #2127
+  - Persistent Stream Provider initialization timeout fix. #2065
+  - Fix null reference in StreamPubSub grain. #2040
+  - Fix for Comment in configuration XML causes exception to be thrown #1994
+  - Fixed null MethodInfo in Interceptors #1938
+  - Object Pools not pooling fix. #1937 
+  - Fixed EventHub stream provider flow control. #1935
```
Comment (jdom) [inline changelog.md]: where these bugs ever introduced in an official release? otherwise we should just not mention them

File: changelog.md [hunk 8]
```diff
@@ -4,20 +4,116 @@ All notable end-user facing changes are documented in this file.
...
+  - Harden explicit subscription pubsub system #1884
+  - Retry EventHub receiver initialization on failure. #1922
+  - Begin reading from most recent EventHub data using latest offset #1921
+  - Fix #1869. Grain Extensions + method interception should function correctly #1874
+  - Fix bug with generic state parameter caused by inconsistent use of grainClassName / genericArgument / genericInterface #1897
+  - Fixes to the default ServiceProvider and back compat with grain creation #1911
+  - update blob storage provider to throw on storage exceptions #1902
+  - Fix several incorrect usages of Type and TypeInfo #1894
```
Comment (jdom) [inline changelog.md]: Not user facing

File: changelog.md [hunk 9]
```diff
@@ -4,20 +4,116 @@ All notable end-user facing changes are documented in this file.
...
+  - Fixed null MethodInfo in Interceptors #1938
+  - Object Pools not pooling fix. #1937 
+  - Fixed EventHub stream provider flow control. #1935
+  - Harden explicit subscription pubsub system #1884
+  - Retry EventHub receiver initialization on failure. #1922
+  - Begin reading from most recent EventHub data using latest offset #1921
+  - Fix #1869. Grain Extensions + method interception should function correctly #1874
+  - Fix bug with generic state parameter caused by inconsistent use of grainClassName / genericArgument / genericInterface #1897
+  - Fixes to the default ServiceProvider and back compat with grain creation #1911
+  - update blob storage provider to throw on storage exceptions #1902
```
Comment (jdom) [inline changelog.md]: I would put this outside of bug fixes, since it's a breaking change

File: changelog.md [hunk 10]
```diff
@@ -4,20 +4,116 @@ All notable end-user facing changes are documented in this file.
...
+  - Fix several incorrect usages of Type and TypeInfo #1894
+  - Stacktrace preservation when sending exception to a remote process. #1879
```
Comment (jdom) [inline changelog.md]: This was already in 1.2.3 I believe

File: changelog.md [hunk 11]
```diff
@@ -4,20 +4,116 @@ All notable end-user facing changes are documented in this file.
...
-- Removed OrleansDependencyInjection package and instead Orleans references Microsoft.Extensions.DepedencyInjection #1911 #1901 #1878
-  - Now using Microsoft.Extensions.DepedencyInjection.ServiceProvider as the default service provider if the user does not override it.
-  - Grains are still not being injected automatically unless the user opts in by specifying his own Startup configuration that returns a service provider.
-- Updated Azure Storage dependency to 7.0.0 #1968
-- A new ADO.NET storage provider that is significantly easier to setup, which replaces the the previous one. This change is not backwards compatible and does not support sharding
+- Invoker codegen: methods returning Task<object> do not need Box() calls #2221
+- CodeGen: Avoid wrapping IGrainMethodInvoker.Invoke body in try/catch #2220
+- Several major performance improvements
+### [v1.3.0-beta2]
+- Support for geo-distributed multi-cluster deployoments #1108 #1109 #1800
```
Comment (dVakulen) [inline changelog.md]: Typo: deployments

File: changelog.md [hunk 12]
```diff
@@ -4,20 +4,116 @@ All notable end-user facing changes are documented in this file.
...
+- Reworked documentation to use DocFX #1970
+- Providers
+  - Support for PostgreSql #2113
+  - Remove confusing parameter from AzureSilo.Start #2109
+  - Minimal Service Fabric integration #2120
+  - Decode protobuff using MessageParser, not dynamic #2136
```
Comment (dVakulen) [inline changelog.md]: Nit: redundant second 'f' in  'protobuff' word.

File: changelog.md (hunk 11 above)
Comment (sergeybykov) [inline changelog.md]: Thanks. Fixed.

**Pull request merged**